README.md

```diff
@@ -0,0 +1,71 @@
+# Flocx-ui
+
+Flocx plugin for horizon to show and create bids, offers, and contracts
+
+* Free software: Apache license
+* Release Notes: View the [changelog](CHANGELOG.md)
+* Contributing: Read [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## Enabling in DevStack
+
+Add this repo as an external repository into your `local.conf` file:
+
+```conf
+[[local|localrc]]
+enable_plugin flocx https://github.com/CCI-MOC/flocx-ui
+```
+
+## Manual Installation
+
+The following below instructions assumes that Horizon is already installed and its installation folder is `<horizon>`. Detailed information on how to install Horizon can be found at [https://docs.openstack.org/horizon/latest/contributor/quickstart.html#setup](https://docs.openstack.org/horizon/latest/contributor/quickstart.html#setup).
+
+The installation folder of Flocx-ui will be referred to as `<flocx-ui>`.
+
+Flocx-ui uses [Node.js](https://nodejs.org) to run linters and JavaScript unit tests. Make sure to install Node.js before continuing.
+
+---
+
+Clone Flocx-ui
+
+```bash
+$ git clone https://github.com/CCI-MOC/flocx-ui
+$ cd flocx-ui
+```
+
+Install requirements
+
+```bash
+$ npm install # This should install python dependencies as well
+```
+
+Install source code
+
+```bash
+$ sudo python setup.py install
```

Comment:
We shouldn't require (or suggest) `sudo` here. Ideally, anything python can be installed either (a) into a virtual environment or (b) into `~/.local` using `pip install --user`.

Reply:
I also thought it was strange that `sudo` was used here. For context, I was following [another horizon plugin](https://github.com/openstack/congress-dashboard)’s `README.md` as a template for this and so I just assumed that it was necessary.

Reply:
@larsks I’m also realizing now that extra files are needed for the “Enabling in DevStack” section, so I should probably remove that until I actually add those files. While I am making that change, would you like me to remove the sudos or leave them there?